### DISCUSSION TITLE
How to stop .thumb.jpg generation?

### DISCUSSION THREAD

**Discussion by Pluuskie** [vladmandic/sdnext#1632] [Q&A] [answered]
Anyone know how to stop the creation of thumbnail images? I've already setup custom thumbnails/preview images and they keep popping back in after I delete them and it clutters up my extra network folders more than they already are.

**Comment by Pluuskie** [comment]
Ok I might've misunderstood what the .thumb files were for, my bad.

**Comment by vladmandic** [comment, marked as answer]
no worries. but this did give me idea, if your previews are already within reasonable resolution range, there is no need to generate thumbnails, so now its going to be that thumbnail is generate from preview only if preview resolution is above 1k.
